docs/concepts/cluster-administration/logging.md

```diff
@@ -61,13 +61,28 @@ Everything a containerized application writes to `stdout` and `stderr` is handle
 
 By default, if a container restarts, the kubelet keeps one terminated container with its logs. If a pod is evicted from the node, all corresponding containers are also evicted, along with their logs.
 
-An important consideration in node-level logging is implementing log rotation, so that logs don't consume all available storage on the node. Kubernetes uses the [`logrotate`](http://www.linuxcommand.org/man_pages/logrotate8.html) tool to implement log rotation.
-
-Kubernetes performs log rotation daily, or if the log file grows beyond 10MB in size. Each rotation belongs to a single container; if the container repeatedly fails or the pod is evicted, all previous rotations for the container are lost. By default, Kubernetes keeps up to five logging rotations per container.
-
-The Kubernetes logging configuration differs depending on the node type. For example, you can find detailed information for GCI in the corresponding [configure helper](https://github.com/kubernetes/kubernetes/blob/{{page.githubbranch}}/cluster/gce/gci/configure-helper.sh#L96).
-
-When you run [`kubectl logs`](/docs/user-guide/kubectl/v1.6/#logs), as in the basic logging example, the kubelet on the node handles the request and reads directly from the log file, returning the contents in the response. Note that `kubectl logs` **only returns the last rotation**; you must manually extract prior rotations, if desired and cluster-level logging is not enabled.
+An important consideration in node-level logging is implementing log rotation,
+so that logs don't consume all available storage on the node. Kubernetes
+in version less than or equal to 1.7 is not responsible for rotating logs,
+but rather a deployment tool should set up a solution to address that.
+For example, in Kubernetes clusters, deployed by the `kube-up.sh` script,
+there is a [`logrotate`](http://www.linuxcommand.org/man_pages/logrotate8.html)
+tool configured to perform log rotations daily or once an application's log
+file grows beyond 10MB. You can also set up a container runtime to rotate logs
+automatically, e.g. by using Docker's `log-opt`. As an example, you can find
```

Comment:
In COS on GCP, yes. Agree, considering also the link below, this part may be confusing. I'll address it, thanks